style.css

```diff
@@ -0,0 +1,717 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 auto;
+  padding: 0 6%;
+  font-family: Gotham Pro, Arial, serif;
+  width: auto;
+  max-width: 1170px;
+}
+
+.navigation {
+  margin-bottom: 19px;
+  height: 95.79px;
+}
+
+.navigation__logo {
+  display: none;
+}
+
+.navigation__banner {
+  margin: 0;
+  padding-top: 30px;
+  padding-bottom: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.navigation__menu {
+  display: flex;
+  justify-content: flex-start;
+  flex: 3;
+}
+
+.navigation__link {
+  margin-right: 6.7%;
+}
+
+.navigation__link_s {
+}
+
+.navigation__form {
+  padding-top: 7px;
+  border-bottom: 1px solid #000;
+  margin-right: 2%;
+  width: 20%;
+  flex: 1;
+}
+
+.navigation__input {
+  padding: 0;
+  border: none;
+  outline: none;
+  vertical-align: middle;
+  width: 88%;
+}
+
+.navigation__search {
+  width: 9%;
+  padding: 0;
+  border: none;
+  background: url(images/search.png) no-repeat 100%;
+}
+
+.navigation__submenu {
+  display: flex;
+  justify-content: space-around;
+  flex: 1;
+  margin: 2px 0.9% 0 0;
+}
+
+.navigation__img {
+  margin-right: 4.4%;
+  padding-top: 1px;
+}
+
+.navigation__link span {
+  display: inline-block;
+  border: none;
+  color: #FFF;
+  font-family: Gotham Pro, Arial, serif;
+  font-size: 10px;
+  font-weight: bold;
+  text-align: center;
+  line-height: 14.92px;
+  background: #000;
+  width: 14.92px;
+  height: 14.92px;
+  border-radius: 50%;
+  padding: 0;
+}
+
+.navigation__us {
+  width: 20px;
+  height: 20px;
+}
+
+.navigation__burger {
+  display: none;
+}
+
+a {
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: bold;
+  color: #000;
+}
+
+.video__main, 
+.stories__text {
+  font-size: 14px;
+  line-height: 17px;
+}
+
+.catalog__wrapper {
+  margin-bottom: 49px;
+  padding: 89px 0 0 60px;
+  width: 100%;
+  height: 280px;
+  background: url(images/background.jpg) no-repeat;
+  background-size: cover;
+  color: #FFF;
+}
+
+.catalog__title {
+  margin: 0;
+  padding-bottom: 12px;
+  width: 25%;
+  font-size: 30px;
+  line-height: 29px;
+}
+
+.catalog__shop {
+  margin: 0;
+}
+
+.catalog__shop a {
+  line-height: 11px;
+  color: #FFF;
+}
+
+a:hover { color: #F33; }
+
+a img:hover { background: #F33; }
+
+button:hover { 
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Project.Axion/pullRequest?prid=5456827)